linera-execution/src/bin/wit_generator.rs

```diff
@@ -142,34 +128,51 @@ impl Operation for WriteToFile {
 /// Checks that a WIT file has the expected contents.
 pub struct CheckFile;
 
-impl Operation for CheckFile {
-    fn run_for_file<'c>(
-        &mut self,
-        path: &Path,
-        contents: impl Iterator<Item = &'c str>,
-    ) -> Result<()> {
-        let mut buffer = Vec::new();
-        let mut file = BufReader::new(
-            File::open(path)
-                .with_context(|| format!("Failed to open file at {}", path.display()))?,
-        );
+struct FileComparer {
```

Comment:
Nit: rename to `FileComparator`?

Reply:
Yes that's fair. "Comparer" is not even a word apparently.